omni-reader/README.md

```diff
@@ -0,0 +1,224 @@
+# OmniReader - Multi-model text extraction comparison
+
+OmniReader is a document processing workflow that ingests unstructured documents (PDFs, images, scans) and extracts text using multiple OCR models. It provides side-by-side comparison of extraction results, highlighting differences in accuracy, formatting, and content recognition. The multi-model approach allows users to evaluate OCR performance across different document types, languages, and formatting complexity. OmniReader delivers reproducible, automated, and cloud-agnostic analysis, with comprehensive metrics on extraction quality, processing time, and confidence scores for each model. It also supports parallel processing for faster batch operations and can compare an arbitrary number of models simultaneously.
+
+## 🚀 Getting Started

```

Comment:
before we jump in im missing the "Why". why this vs any other product that does it. who should start this pipeline and use this app. E.g. ML Engineers who want to run their own batch OCR pipelines?